lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -397,8 +410,13 @@ def create(self, tblname, idxtype, columns, cond, tblspace, storage_params, conc
         if idxtype is None:
             idxtype = "BTREE"
 
-        query = 'CREATE INDEX'
-
+        query = 'CREATE'
+        
+        if unique:
```

Comment:
Would be also great to check `idxtype` here. Currently only b-tree supports unique indexes

Reply:
Nice catch @kostiantyn-nemchenko I'll update it